sample_data/sample_config.ini

```diff
@@ -56,6 +56,11 @@ target_vocabulary_type=word
 source_vocabulary_size=4100
 target_vocabulary_size=4900
 
+; If you specified the model directory here, load vocabulary from the specified model.
+; If you use this option, the vocabulary type and the vocabulary size specified above will be ignored.
+; By writing vocabulary_model=none, this option will be ignored.
+vocabulary_model=none
```

Comment:
This allows us not to load vocabulary from `none` directory. It might be better to separate switch and path options.